src/components/main/compose/messages/mod.rs

```diff
@@ -321,11 +321,15 @@ pub fn Messages(cx: Scope<Props>) -> Element {
     );
 
     let rg = cx.props.messaging.clone();
-    let senders: Vec<DID> = messages.iter().map(|msg| msg.sender()).collect();
-    // messages has already been reversed
-    let idx_range = 0..messages.len();
-    let next_sender = idx_range.clone().map(|idx| senders.get(idx + 1));
-    let prev_sender = idx_range.map(|idx| if idx == 0 { None } else { senders.get(idx - 1) });
+    let senders: Vec<DID> = match current_chat {
```

Comment:
This could probably be done by doing 

```rust
let senders = current_chat.map(|info| info.conversation.recipients()).unwrap_or_default();
```

which should also correct the clippy warning there.